python/lsst/ts/tunablelaser/mock_server.py

```diff
@@ -131,68 +181,194 @@ def parse_message(self, msg):
             raise Exception
 
     def do_wavelength(self):
+        """Return current wavelength.
+
+        Returns
+        -------
+        `bytes`
```

Comment:
This is a string, not a bytes (which I think is great).

Also please make it clear that the returned data is a formatted str, not, say, a float (which is what I assumed from reading the doc string).

Here is one possibility:
```
"""Return current wavelength as a formatted str."""
```

Please fix the doc strings for the other do_ methods, as well.